engine/engine.gyp

```diff
@@ -289,12 +289,26 @@
 						
 						# Forces all dependencies to be linked properly
 						'type': 'shared_library',
+                        
+						'conditions':
+						[
+							[
+								'("11" not in target_sdk) and ("12" not in target_sdk) and ("13" not in target_sdk)',
```

Comment:
@livecodepanos if you want to do a numeric comparison you can use 'float(target_sdk) >= 14.0' instead

Reply:
@livecodeian `target_skd` does not only contain the version number but also it contains the platform name. So it can be e.g. `iphoneos14.5` or `iphonesimulator14.5`.

Is there a neater way to say "if version >= 14", given the above?